spec/system/composer/category_templates_spec.rb

```diff
@@ -24,6 +24,34 @@
   fab!(:form_template_4) do
     Fabricate(:form_template, name: "Biography", template: "- type: textarea")
   end
+  fab!(:form_template_5) do
+    Fabricate(
+      :form_template,
+      name: "Medication",
+      template:
+        %Q(
+        - type: input\n
+          attributes:\n
+            label: \"What is your name?\"\n
+            placeholder: \"John Smith\"\n
+          validations:\n
+            required: false\n
+        - type: upload\n
+          attributes:\n
+            file_types: \".jpg, .png\"\n
+            allow_multiple: false\n
+            label: \"Upload your prescription\"\n
+            validations:\n
+            required: true\n
+        - type: upload\n
+          attributes:\n
+            file_types: \".jpg, .png, .pdf, .mp3, .mp4 \"\n
+            allow_multiple: true\n
+            label: \"Any additional docs\"\n
+            validations:\n
+            required: false"),
```

Comment:
```suggestion
<<-HEREDOC
  - type: input
    attributes:
      label: "What is your name?"
      placeholder: "John Smith"
    validations:
      required: false
  - type: upload
    attributes:
      file_types: ".jpg, .png"
      allow_multiple: false
      label: "Upload your prescription"
    validations:
      required: true
  - type: upload
    attributes:
      file_types: ".jpg, .png, .pdf, .mp3, .mp4"
      allow_multiple: true
      label: "Any additional docs"
    validations:
      required: false
HEREDOC
```

Is it possible to do something like this to avoid having to manually add '\n'?

Reply:
I just pushed an update removing the `/n` and escaped characters, but maintained the `%Q()` syntax as the auto-formatter messes things up and causes things to break when I use `<<-HEREDOC` for some reason.